examples/01_image.py

```diff
@@ -14,6 +14,8 @@
 
 import viser
 
+from viser._message_api import TitlebarConfig, TitlebarPadding, TitlebarImage, TitlebarButton
```

Comment:
maybe we can expose these in `viser.theme` instead of the private `_message_api` namespace